financial-connections/src/main/java/com/stripe/android/financialconnections/features/consent/ConsentViewModel.kt

```diff
@@ -58,18 +60,23 @@ internal class ConsentViewModel @Inject constructor(
     private fun logErrors() {
         onAsync(
             ConsentState::consent,
-            onSuccess = { eventTracker.track(FinancialConnectionsEvent.PaneLoaded(Pane.CONSENT)) },
+            onSuccess = { eventTracker.track(PaneLoaded(Pane.CONSENT)) },
             onFail = { logger.error("Error retrieving consent content", it) }
         )
         onAsync(ConsentState::acceptConsent, onFail = {
-            eventTracker.track(Error(Pane.CONSENT, it))
-            logger.error("Error accepting consent", it)
+            eventTracker.logError(
+                extraMessage = "Error accepting consent",
+                error = it,
+                logger = logger,
+                pane = Pane.CONSENT
+            )
```

Comment:
unified some error logs into `eventTracker.logError`, that will be used to also log errors externally via the new event listener.